domains/authorization/Authorization.API/API/Authorization/[Features]/DeleteConsentCommand.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using API.Data;
+using API.Repository;
+using API.ValueObjects;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+namespace API.Authorization._Features_;
+
+public class DeleteConsentCommandHandler(
+    IConsentRepository consentRepository,
+    IUserRepository userRepository,
+    IUnitOfWork unitOfWork)
+    : IRequestHandler<DeleteConsentCommand, bool>
+{
+    public async Task<bool> Handle(DeleteConsentCommand request, CancellationToken cancellationToken)
+    {
+        await unitOfWork.BeginTransactionAsync();
+
+        var consent = await consentRepository.Query()
+            .Include(c => c.Organization)
+            .FirstOrDefaultAsync(c => c.ClientId == request.ClientId && c.OrganizationId == request.OrganizationId, cancellationToken);
+
+        if (consent == null)
```

Comment:
You can throw EntityNotFoundException here. This will automatically create a 404 not found. 
Then you can avoid that part in the controller

Reply:
Fixed: https://github.com/Energinet-DataHub/energy-origin/pull/1236/commits/94b1f55c3688b382e42989a6820e5bde220f891d